### PR TITLE
PWA updates

### PR DESCRIPTION
**Pull request by dodumosu**
fixes nditech/apollo-issues#114

### CODE REVIEW

**Comment by dodumosu** [comment]
also addresses nditech/apollo-issues#116 as far as submissions on iOS is concerned

**Comment by takinbo** [comment]
@dodumosu can we rename this PR appropriately?

**Comment by takinbo** [comment]
Could you also rebase and resolve the conflicts?

File: apollo/settings.py [hunk 1]
```diff
@@ -41,6 +41,8 @@
 
 SENTRY_DSN = config('SENTRY_DSN', default=None)
 SENTRY_USER_ATTRS = ['email']
+TRACK_PWA_ERRORS = config(
```
Comment (takinbo) [inline apollo/settings.py:44]: Can we avoid having a PWA specific error tracking setting? If `DEBUG` is true, then we should track PWA errors.

File: apollo/pwa/static/js/app.js [hunk 2]
```diff
@@ -1,33 +1,29 @@
...
-                                break;
-                        }
-                    };
-                };
-            })
-            .catch(error => console.error('service worker registration failed:', error));
+  if ('serviceWorker' in navigator) {
+    navigator.serviceWorker.register('/pwa/serviceworker.js', {scope: '/pwa/'})
+      .then(registration => {
+        console.log('service worker registered successfully with scope:', registration.scope);
```
Comment (takinbo) [inline apollo/pwa/static/js/app.js:5]: we want to remove these "debug" statements when the application gets to production

**Review by takinbo — APPROVED** [review]

**Pull request merged**